packages/wallets/src/components/DerivAppsSection/__tests__/DerivAppsGetAccount.spec.tsx

```diff
@@ -64,8 +67,10 @@ const wrapper = ({ children }: PropsWithChildren) => {
 
 describe('DerivAppsGetAccount', () => {
     beforeEach(() => {
+        (useDevice as jest.Mock).mockReturnValue({ isDesktop: true });
         jest.clearAllMocks();
```

Comment:
```suggestion
        jest.clearAllMocks();
        (useDevice as jest.Mock).mockReturnValue({ isDesktop: true });
```

Shouldn't `(useDevice as jest.Mock).mockReturnValue({ isDesktop: true });` come after `jest.clearAllMocks();` @aizad-deriv? It looks like you're setting it, only to clear it. 🤔